decidim-proposals/app/views/decidim/proposals/admin/proposals/show.html.erb

```diff
@@ -49,32 +49,34 @@
             </p>
           </div>
         </div>
-        <div class="card-section">
-          <div class="row column">
-            <span class="component__show-title"><%= t ".endorsers" %></span>
-            <ul id="proposal-endorsers-list">
-              <% presented_endorsers = endorsers_presenters_for(proposal).to_a %>
-              <% presented_endorsers.first(5).each do |presented_endorser| %>
-                <li>
-                  <%= link_to_if(
-                        presented_endorser.profile_path.present?,
-                        presented_endorser.name,
-                        presented_endorser.profile_path,
-                        target: :blank
-                      ) %>
-                </li>
-              <% end %>
-              <% if presented_endorsers.count > 5 %>
-                <li>
-                  <%= link_to(
-                        t(".n_more_endorsers", count: presented_endorsers.count - 5),
-                        resource_locator(proposal).path
-                      ) %>
-                </li>
-              <% end %>
-            </ul>
+        <% presented_endorsers = endorsers_presenters_for(proposal).to_a %>
+        <% if presented_endorsers.any? %>
+          <div class="card-section">
```

Comment:
I am thinking maybe is worth extracting to a partial ?

Reply:
I agree, but let's do it in another PR as a refactor. As it is now I think would be easier to backport to v0.27 and v0.26